pkgs/applications/networking/p2p/tribler/default.nix

```diff
@@ -1,13 +1,20 @@
 { stdenv, fetchurl, python3Packages, makeWrapper
-, enablePlayer ? true, libvlc, qt5, lib }:
+, libtorrent-rasterbar-1_2_x
+, enablePlayer ? true, qt5, lib }:
 
+let
+  python = python3Packages.python;
+
+  libtorrent = (python3Packages.toPythonModule (
+    libtorrent-rasterbar-1_2_x.override { inherit python; })).python;
+in
 stdenv.mkDerivation rec {
   pname = "tribler";
-  version = "7.4.4";
+  version = "7.10.0";
 
   src = fetchurl {
     url = "https://github.com/Tribler/tribler/releases/download/v${version}/Tribler-v${version}.tar.xz";
-    sha256 = "0hxiyf1k07ngym2p8r1b5mcx1y2crkyz43gi9sgvsvsyijyaff3p";
+    sha256 = "03pav0fi8rxmbwgn2ywd281wx5dwja1w0ds3wgzagpd7nk8iw6pk";
```

Comment:
I'm getting a different hash:

```
$ nix-prefetch-url https://github.com/Tribler/tribler/releases/download/v7.10.0/Tribler-v7.10.0.tar.xz
[3.2 MiB DL]
path is '/nix/store/y7pd7g4011bxxpv4libwis6l9vkajlja-Tribler-v7.10.0.tar.xz'
1x45z23d1cqf0lai7wg5ki7gi2vba5hqk0swhggzplcjwma4wmh9
```

Reply:
OH. I also get yours today. Maybe they switched it upstream?

Here is the day when I got the hash: https://github.com/viric/nixpkgs/commit/53fabb2b18ded07515c7932747dcea5fa40ae34f so it is not that I forgot the rc2 hash.

Reply:
I have the tarballs in the store. It seems I picked their tarball right after the release, because they changed the tarball 12 minutes later.